flo_telepresence/src/display_cv.py

```diff
@@ -257,29 +263,42 @@ def __draw_home(self):
                         (255, 255, 255))
             cv2.putText(self.filled_home,
                         'WiFi Quality: {:.1f}'.format(self.wifi_quality),
-                        (10, 342),
+                        (10, 323),
                         self.font,
                         1,
                         (255, 255, 255))
             cv2.putText(self.filled_home,
                         'WiFi Signal: {:.1f} dB'.format(self.wifi_signal),
-                        (400, 342),
+                        (400, 323),
                         self.font,
                         1,
                         (255, 255, 255))
             cv2.putText(self.filled_home,
                         'Server: '+self.server_addr,
-                        (10, 406),
+                        (10, 368),
                         self.font,
                         1,
                         (255, 255, 255))
             connected = self.connected_clients > 0
             cv2.putText(self.filled_home,
                         'Connected' if connected else 'Not Connected',
-                        (400, 406),
+                        (400, 368),
                         self.font,
                         1,
                         (0, 255, 0) if connected else (0, 0, 255))
+            cv2.putText(self.filled_home,
+                        'Recording' if self.recording else 'Not Recording',
+                        (10, 413),
+                        self.font,
+                        1,
+                        (0, 255, 0) if self.recording else (0, 0, 255))
```

Comment:
flip colors. Recording when on home screen is bad, should be red.

Reply:
Yup I'll make this change